frontend/src/main/web/src/components/NavItem.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react' // eslint-disable-line
+import { Link, NavIcon } from './'
+import { merge } from 'lodash'
+
+const NavItem = ({
+  id,
+  link,
+  small,
+  icon,
+  active,
+  title,
+  useHref,
+  ...props
+}) => {
+  const logo = (icon === 'zanata')
+  const search = (link === '/search')
```

Comment:
Names could communicate that these are booleans: `isLogo`, `isSearchLink` (especially since usage is so far below).